packages/components/src/components/Icon/Icon.tsx

```diff
@@ -60,7 +70,9 @@ const Icon = React.forwardRef<HTMLDivElement, IconProps>(
           aria-label={title}
           as={RenderedIcon}
           display="block"
+          h={size}
           verticalAlign="middle"
+          w={size}
```

Comment:
Just for my understanding, why was this change necessary?

Reply:
if we don't apply those, lucide icons will have a default value of 24 fitting the container

![Screen Shot 2023-05-19 at 15 29 12](https://github.com/Localitos/pluto/assets/2047941/9ef82135-bc5a-4dd3-b393-ab19422d6f82)